File: src/test/java/boat/utilities/PropertiesHelperTest.java
package boat.utilities;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertNotNull;
import static org.junit.jupiter.api.Assertions.assertTrue;

class PropertiesHelperTest {

    @Test
    void getVersion() {
        // Given
        // When
        String version = PropertiesHelper.getVersion();
        // Then
        assertTrue(!version.contains("version-missing"));
    }

    @Test
    void getProperty() {
        // Given
        // When
        String property = PropertiesHelper.getProperty("RCLONEDIR");
        // Then
        assertNotNull(property);
        assertTrue(property.length() > 0);
    }
}

File: src/test/java/boat/utilities/PropertiesHelperTest.kt
package boat.utilities

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class PropertiesHelperTest {

    @Test
    fun version() {
        // Given
        // When
        val version = PropertiesHelper.version
        // Then
        Assertions.assertNotNull(version)
        Assertions.assertTrue(!version!!.contains("version-missing"))
    }

    @Test
    fun property() {
        // Given
        // When
        val property = PropertiesHelper.getProperty("PATH")
        // Then
        Assertions.assertNotNull(property)
        Assertions.assertTrue(property!!.isNotEmpty())
    }
}